src/connectors/autocomplete/connectAutocomplete.ts

```diff
@@ -59,14 +59,18 @@ export type AutocompleteConnector = Connector<
   AutocompleteConnectorParams
 >;
 
+type WidgetParams = {
+  escapeHTML?: boolean;
+};
+
 const connectAutocomplete: AutocompleteConnector = function connectAutocomplete(
   renderFn,
   unmountFn = noop
 ) {
   checkRendering(renderFn, withUsage());
 
-  return widgetParams => {
-    const { escapeHTML = true } = widgetParams || ({} as typeof widgetParams);
+  return (widgetParams: WidgetParams) => {
```

Comment:
isn't this `AutocompleteConnectorParams` already?

Reply:
was experimental. I've reverted it back to the original.